.github/scripts/python/release-bundle.py

```diff
@@ -0,0 +1,435 @@
+    parser.add_argument("--release_bundle_version", action="store",help="release bundle version")
+    parser.add_argument("--signing_key_name",action="store",help="release bundle signing key name")
+    parser.add_argument("--arti_host", action="store", help="artifactory base url")
+    parser.add_argument("--release_method", action="store", help="source type to create release bundle")
+
+    args = parser.parse_args()
+
+    build_name = args.build_name  # for rt buildinfo query
+    build_number = args.build_number  # for rt buildinfo query
+    build_version = args.build_version
+    rt_auth = (args.rt_auth_username, args.rt_auth_password)
+    manifest_file_path = args.manifest_file_path
+    rt_base_url = args.rt_base_url
+    jf_cli_rt_name = args.jf_cli_rt_name
+    logging_level = args.logging_level
+    dry_run = args.dry_run
+    relese_bundle_name=args.release_bundle_name
+    release_bundle_version=args.release_bundle_version
+    signing_key_name=args.signing_key_name
+    arti_host=args.arti_host
+    release_method=args.release_method
+
+    ############ End parsing args #############
+
+    ######### logging ############
+    string_to_level = {
+        "DEBUG": logging.DEBUG,
+        "INFO": logging.INFO,
+        "WARNING": logging.WARNING,
+        "ERROR": logging.ERROR,
+        "CRITICAL": logging.CRITICAL
+    }
+
+    logging.basicConfig(
+        level=string_to_level[logging_level],
+        format='[%(asctime)s] [%(filename)s:%(lineno)d] %(levelname)s - %(message)s',
+        datefmt='%H:%M:%S'
+    )
+    
+    ####### End of logging #######
+
+    ####### Upload to box ##########
+    # determine the build suffix 9.5.1.0-124
+
+    build_suffix = build_version + '-' + build_number
+    
+    if release_method == "artifacts":
+        # downloads artifacts
+        builds_output_json, artifacts_in_build_info = get_artifact_info_json(build_name, build_number, rt_auth=rt_auth,
+                                                                            rt_base_url=rt_base_url,
+                                                                            jf_cli_rt_name=jf_cli_rt_name)
+        file_folder_dict = process_manifest_yaml(get_manifest_yaml(build_suffix, manifest_file=manifest_file_path))
+        artifacts_to_release,artifacts_to_release_details = get_manifest_buildinfo_intersect(file_folder_dict, builds_output_json)
+        
+        # if there are no files to deploy, exit process
+        if not artifacts_to_release:
+            logging.warning(
+                f'There are no artifacts to be promoted with build name:{build_name}, build number: {build_number}, build '
+                f'version {build_version}')
+            sys.exit(1)
+            
+        
+        create_release_bundle_from_artifacts( artifacts_to_release_details, relese_bundle_name, release_bundle_version, signing_key_name, rt_auth=rt_auth, arti_host=arti_host )
+   
+    if release_method == "builds":
```

Comment:
Why isn't this an `else`? Is there the possibility for `release_method` to have multiple values?

Reply:
It can have multiple values like source type as AQL query, Builds, Artifacts ,release bundles

Reply:
But is the `release_method` to have the values `artifacts` and `build` at the same time? I don't think so - the 
release_method` is either `artifacts` or `builds`. This `if` should be an `else if ` at least!

Reply:
Ok, I will add eles if statement.